doc/getting_started_rtkernel.rst

```diff
@@ -80,6 +66,18 @@ following to create and configure the build::
     -DCMAKE_ECLIPSE_EXECUTABLE=/opt/rt-tools/workbench/Workbench \
     -DCMAKE_ECLIPSE_GENERATE_SOURCE_PROJECT=TRUE -G "Eclipse CDT4 - Unix Makefiles"
 
+If you instead is using Windows, start an MSYS session, such as the program 
+"Command line" found in the rt-labs rt-collab Toolbox installation 
+(adapt paths to your setup)::
+
+    cd /c/Users/rtlfrm/Documents/Profinet/build_rtkernel
```

Comment:
This is unnecessarily complicated. The commands above should work fine in the Toolbox Command Line. It does for me anyway; are you seeing some issues? 

It's good to mention that the Toolbox Command Line should be used though, this used to be in the README but disappeared along the way.

Reply:
Actually now I did find a problem. The cmake invocation shown uses --B (two hyphens) but should use -B (one hyphen). Maybe you could fix that in this PR?

Reply:
That --B is changed in my documentation PR #232

Reply:
cmake (without specifying its full path) does not work as cmake is not present in the MSYS PATH variable.